centreon/tests/e2e/features/Api-Token/05-api-token-acl-rights.feature

```diff
@@ -0,0 +1,19 @@
+@ignore
+@REQ_MON24872
+Feature: ACL Permissions for Administrators
+
+  As an Administrator
+  I want to have specific access rights in the ACL menu
+  So that I can manage organization tokens and access the API token page
+
+  Background:
+    Given I am logged in as an Administrator
+    And I am on the ACL menu
+
+  Scenario: Verify "Manage Organization Token" Menu Action in ACL
```

Comment:
Is the goal of the scenario really is "Verify "Manage Organization Token" Menu Action in ACL"
or is more like "Restrict access to Manage Organization Token action"